fi/features.html

```diff
@@ -0,0 +1,458 @@
+                    <a href="http://www.sympygamma.com/input/?i=exp%28x%29+%2F+%281+%2B+exp%282x%29%29">eksponentit</a>,
+                    <a href="http://www.sympygamma.com/input/?i=root%28x%2C+7%29+**+2">juuret</a>,
+                    <a href="http://www.sympygamma.com/input/?i=log%28-25%29">logaritmit</a>,
+                    <a href="http://www.sympygamma.com/input/?i=Abs%281+%2B+3I%29">itseisarvot</a>,
+                    <a href="http://www.sympygamma.com/input/?i=Ylm%281%2C+-1%2C+theta%2C+phi%29">harmoniset pallofunktiot</a>,
+                    <a href="http://www.sympygamma.com/input/?i=gamma%2841%2F2%29+%2B+16%21%21">kertomat ja gammafunktiot</a>,
+                    <a href="http://www.sympygamma.com/input/?i=zeta%284%29">zeta-funktiot</a>,
+                    <a href="http://www.sympygamma.com/input/?i=x**7+-+x**5+%2B+4x**3+%2B+1">polynomit</a>,
+                    <a href="http://live.sympy.org/?evaluate=from%20sympy.abc%20import%20n%2Ca%2Cb%2Cx%0A%23--%0Ajacobi(0%2C%20a%2C%20b%2C%20x)%0A%23--%0Ajacobi(1%2C%20a%2C%20b%2C%20x)%0A%23--%0Ajacobi(2%2C%20a%2C%20b%2C%20x)%0A%23--%0Ajacobi(n%2C%20a%2C%20a%2C%20x)%0A%23--%0Ajacobi(n%2C%20a%2C%20b%2C%200)%0A%23--%0Aconjugate(jacobi(n%2C%20a%2C%20b%2C%20x))%0A%23--%0Aerf(I*oo)%0A%23--%0ADiracDelta(x**2%20%2B%20x%20-%202).simplify(x)%0A%23--%0A">erikoisfunktiot</a>, ...
+                </li>
+                <li><strong>Sijoitus:</strong> <a href="http://www.sympygamma.com/input/?i=%28exp%28x%29+%2F+%281+%2B+exp%282x%29%29%29.subs%28exp%28x%29%2C+u%29">esimerkki</a></li>
+                <li><strong>Numerot:</strong>
+                    
+                    <a href="http://www.sympygamma.com/input/?i=100%21">mielivaltaisen tarkat kokonaisluvut</a>,
+                    <a href="http://www.sympygamma.com/input/?i=0.%5B13579%5D">rationaaliluvut</a>,
+                    <a href="http://www.sympygamma.com/input/?i=exp%2820%21%29">ja liukuluvut</a>
+                    
+                </li>
+                <li><strong>Ei-kommutoivat symbolit</strong></li>
+                <li><strong>Kuvion täsmäys</strong></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Polynomit</h3>
+        <div>
+            <p><a href="http://docs.sympy.org/latest/modules/polys/index.html">Dokumentaatio</a></p>
+            <ul>
+                <li><strong>Perusaritmetiikka:</strong> jaollisuus, syt, ...</li>
+                <li><strong>Tekijöihin jakaminen</strong></li>
+                <li><strong>Neliövapaa hajotelma</strong></li>
+                <li><strong>Gröbner-kannat</strong></li>
+                <li><strong>Osittaismurtohajotelma</strong></li>
+                <li><strong>Resultantit</strong></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Differentiaalilaskenta</h3>
+        <div>
+            <p><a href="http://docs.sympy.org/latest/modules/integrals/integrals.html">Integraatiodokumentit</a></p>
```

Comment:
Perhaps 'Integrointidokumentit' would be better. 'Integraatio' tends to mean something else.